python/cugraph/cugraph/tests/test_node2vec.py

```diff
@@ -35,10 +37,20 @@ def setup_function():
     gc.collect()
 
 
+def _get_param_args(param_name, param_values):
+    """
+    Returns a tuple of (<param_name>, <pytest.param list>) which can be applied
+    as the args to pytest.mark.parametrize(). The pytest.param list also
+    contains param id string formed from teh param name and values.
```

Comment:
There is typo.
And nice helper function. Usually we directly pass the list of parameters to @pytest.mark.parametrize for instance like this `DIRECTED = [pytest.param(d) for d in DIRECTED_GRAPH_OPTIONS]`.